components/BacktoTopSection.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react'
+import Scroll from 'react-scroll'
+
+export default class TopArrowSection extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      cssHide: 'button-hide'
+    }
+  }
+
+  componentDidMount () {
+    window.addEventListener('scroll', this.handleScroll)
+  }
+
+  componentWillUnmount () {
+    window.removeEventListener('scroll', this.handleScroll)
+  }
+
+  handleScroll = () => {
+    if (document.body.scrollTop > 20 || document.documentElement.scrollTop > 20) {
+      this.setState({ cssHide: '' })
```

Comment:
Because this function is called on every scrolling, we should call `setState` only when the state really needs to change.

```js
    if (this.state.atTopOfPage) this.setState({ atTopOfPage: false })
```